views/layouts/default.hbs

```diff
@@ -22,42 +22,44 @@
+            <h1><span class="demi">Firefox</span> Monitor</h1>
+          </a>
+        </div>
+        <div class="half">
+          {{> social_media_sharing_buttons}}
+        </div>
+      </header>
+        {{{ body }}}
+      <img id="footer-wave" class="wave" src="/img/wave.svg" />
+    </div>
     <footer id="footer">
       <div class="section-wrapper">
-        <div class="fourth">
-          <a class="footer-logo-wrapper" href="https://www.mozilla.org"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
-          <p>Portions of this content are &copy; 1998-2018 by individual mozilla.org contributors. <br />Content available under a <a href="https://www.mozilla.org/foundation/licensing/website-content/">Creative Commons license</a>.</p>
+        <div class="half">
+          <a class="footer-logo-wrapper" href="https://www.mozilla.org" rel="noopener noreferrer"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
+          <div>
+            <p>Portions of this content are &copy; 1998-2018 by individual mozilla.org contributors. Content available under a <a href="https://www.mozilla.org/foundation/licensing/website-content/" rel="noopener noreferrer">Creative Commons license</a>.</p>
+          </div>
         </div>
         <ul class="fourth">
           <li>Support</li>
-          <li><a href="https://qsurvey.mozilla.com/s3/Firefox-Monitor-Feedback" target="_blank" rel="noopener noreferrer">Give Feedback</a></li>
-          <li><a id="GitHub-logo" href="https://github.com/mozilla/blurts-server/"><img src="/img/github.svg" alt="Github Logo" /></a></li>
-        </ul>
-        <ul class="fourth">
-          <li>Legal</li>
-          <li><a href="https://www.mozilla.org/about/legal/">Terms</a></li>
-          <li><a href="https://www.mozilla.org/privacy/websites/">Privacy</a></li>
+          <li><a href="https://qsurvey.mozilla.com/s3/Firefox-Monitor-Feedback" rel="noopener noreferrer">Give Feedback</a></li>
+          <li><a href="https://www.mozilla.org/about/legal/" rel="noopener noreferrer">Terms</a></li>
+          <li><a href="https://www.mozilla.org/privacy/websites/" rel="noopener noreferrer">Privacy</a></li>
+          <li><a id="GitHub-logo" href="https://github.com/mozilla/blurts-server/" rel="noopener noreferrer"><img src="/img/github.svg" alt="Github Logo" /></a></li>
         </ul>
         <ul class="fourth">
           <li>Firefox</li>
-          <li><a href="https://www.mozilla.org/firefox/new/">Download Firefox</a></li>
-          <li><a href="https://www.mozilla.org/firefox/mobile/">Download Firefox Mobile</a></li>
-          <li><a href="https://www.mozilla.org/firefox/features/">Features</a></li>
-          <li><a href="https://www.mozilla.org/firefox/channel/desktop/">Beta, Nightly, Developer Edition</a></li>
+          <li><a href="https://www.mozilla.org/firefox/new/" rel="noopener noreferrer">Download Firefox</a></li>
```

Comment:
Dang, now that I see this everywhere, I realize that this means mozilla.org's Google Analytics won't get the `Referrer` which will have our `utm_`* params to let them measure our efficacy in downloads and to aDAU.

In general, we don't need to worry as much about sending openers & referrers to other domains that Mozilla controls.

So, let's at least take the `noreferrer` value out of these link `rel`s. Keeping `noopener` should be fine.

Reply:
do we want to throw back in the target="_blank"s